examples/java/src/main/java/org/apache/beam/examples/snippets/transforms/io/gcp/bigquery/BigQueryMyData.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.examples.snippets.transforms.io.gcp.bigquery;
+
+// [START bigquery_my_data]
+
+import com.google.api.services.bigquery.model.TableRow;
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Base64;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.apache.beam.sdk.coders.AvroCoder;
+import org.apache.beam.sdk.coders.DefaultCoder;
+
+class BigQueryMyData {
+
+  static class MyStruct {
+    String stringValue;
+    Long int64Value;
+  }
+
+  @DefaultCoder(AvroCoder.class)
+  static class MyData {
+    String myString;
+    Long myInt64;
+    Double myFloat64;
+    BigDecimal myNumeric;
+    Boolean myBoolean;
+    byte[] myBytes;
+    String myDate;
+    String myDateTime;
+    String myTime;
+    String myTimestamp;
+    String myGeography;
+    List<Long> myArray;
+    MyStruct myStruct;
+
+    public static MyData fromTableRow(TableRow row) {
+      MyData data = new MyData();
+
+      data.myString = (String) row.get("string_field");
+      data.myInt64 = Long.parseLong((String) row.get("int64_field"));
+      data.myFloat64 = (Double) row.get("float64_field");
```

Comment:
My recollection was that sometimes floats come back as strings. I think it depends on whether it is JSON or Avro export.

Reply:
In my tests it always returned as a `Double`, but if it's also possible to return as a `String`, is there a way you would recommend to handle both or should we stick with one?